tests/components/Sidebar-cy.js

```diff
@@ -142,31 +142,30 @@ describe("Sidebar", function() {
       });
     });
 
-    it("renders within the viewport on initial load", function() {
-      cy.get(".sidebar").then(function($sidebar) {
-        const sidebarRect = $sidebar.get(0).getBoundingClientRect();
+    it("Close sidebar when sidebarToggle button clicked", function() {
```

Comment:
Can we also have a test for open sidebar to test the toggle

Reply:
I rather have then running separately (minimum test) that way easier to catch when there is a bug/failure.